slamd/materials/processing/strategies/base_material_strategy.py

```diff
@@ -10,30 +10,33 @@
 
 class BaseMaterialStrategy(ABC):
 
+    @classmethod
     @abstractmethod
-    def create_model(self, submitted_material):
+    def create_model(cls, submitted_material):
         pass
 
+    @classmethod
```

Comment:
Are we sure that the combination classmethod and using inheritance does not lead to any problems?